test/ascii_utils_test.dart

```diff
@@ -23,6 +23,28 @@ void main() {
       }
     });
   });
+
+  group('isOnlyUnderscores', () {
```

Comment:
Consider naming the group to match the getter being tested (`isJustUnderscores`).

Also, there should be a test for an identifier starting with an underscore but followed by non-underscores (`__a`).

Reply:
Oops, done!